tests/test_zipfs.py

```diff
@@ -224,3 +226,38 @@ class TestOpener(unittest.TestCase):
     def test_not_writeable(self):
         with self.assertRaises(NotWriteable):
             open_fs("zip://foo.zip", writeable=True)
+
+
+class FSWithoutMtime(MemoryFS):
```

Comment:
Just being a bit picky, but perhaps this class ought to be named `FSWithoutDetailsNamespace` ?
And perhaps it ought to be a [WrapFS](https://docs.pyfilesystem.org/en/latest/reference/wrapfs.html), rather than an explicit subclass of MemoryFS? :thinking:

Reply:
I've renamed the class, but I don't think that subclassing `WrapFS` (which *composes* filesystems) offers any advantage over subclassing `MemoryFS` in this case.

Reply:
It's https://docs.pyfilesystem.org/en/latest/reference/multifs.html that _composes_ filesystems, but I agree that for this simple test-case, what you've done here is fine.